AdminPortal/src/styles/Login.module.css

```diff
@@ -0,0 +1,147 @@
+.outer {
+    display: flex;
+    height: 600px;
+
+    /*align-items: baseline;*/
+
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+
+    gap: 10.275rem;
+}
+
+.LAAKTAlogo {
+    
+    width: 445.31px;
+    height: 445.31px;
+    margin-top: -40px;
+    /*position: absolute;
+    left: 147px;
+    top: 206px;*/
+}
+
+.formContainer {
+    
+    width: 554.91px;
```

Comment:
For the CSS values, I'd recommend you try to use flexbox where possible instead of hard coding px

Reply:
fixed using vw